main.py

```diff
@@ -1,7 +1,49 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    for genre in ["Western", "Action", "Dramma"]:
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. However, this is corrected later in the code, so it doesn't affect the final result.